helm/cluster-vsphere/templates/cilium-helmrelease.yaml

```diff
@@ -32,6 +32,9 @@ spec:
   values:
     ipam:
       mode: kubernetes
+    k8sServiceHost: api.{{ include "resource.default.name" $ }}.{{ .Values.baseDomain }}
```

Comment:
Does this change mean we have to create DNS records to have a working cilium app? Can't we replace this with an IP somehow? I know IP is assigned later but we can set it like RDE_ID in CAPVCD.

Reply:
Yes it means we require DNS and we do that in all existing providers so far. I wouldn't like to deviate unless we have a general agreement.

Reply:
This means that this chart will not work without having dns-operator in the MC. Not a big issue for us but it can be an issue for external users if there are such people :D

Reply:
I agree. I took a note to update README with a note about DNS. If you want we can discuss this during the kaas sync as well.